### PR TITLE
Makefile: add origin-release target

### PR DESCRIPTION
**Pull request by sttts**
`DOCKER_ORG=sttts make origin-release`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 GOFLAGS :=
+DOCKER_ORG ?= $(USER)
```
Comment (deads2k) [inline Makefile]: I feel like I should hardcode a conversion from `deads` to `deads2k` :)

**Comment by deads2k** [comment]
/lgtm

@sttts mind setting up the other repos?
@ironcladlou our solution

File: Makefile (hunk 1 above)
Comment (soltysh) [reply Makefile]: Sign me up for it too ;)

**Pull request merged**